packages/webapp/src/containers/SensorReadings/saga.js

```diff
@@ -43,157 +43,193 @@ import i18n from '../../locales/i18n';
 
 const sensorReadingsUrl = () => `${sensorUrl}/reading/visualization`;
 
+const getDates = () => {
+  let currentUnixTime = new Date();
```

Comment:
right, I think I lined these all up name-wise because in predicted and historical I was just using this line as a necessary step since date methods are always chainable. I can correct this one.

Reply:
done